### DISCUSSION TITLE
제페토 유니티 패키지 오류입니다.

### DISCUSSION THREAD

**Discussion by YNyum** [naverz/zepeto-studio-kor#794] [5. Q&A]
![제페토 템플릿 오류](https://user-images.githubusercontent.com/53325829/169976223-a42317cd-af57-4dc1-b9e7-77123fa35074.png)
홈페이지 가이드의 안내대로 2020.3.9f1 설치 후 제페토 패키지 improt하는 것까지는 문제 없었으나
에디터와 유니티 허브 재시작 후 템플릿으로 생성 시 오류가 납니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 혹시 방금 발생한 오류일까요?
오후 3시경 일시적인 장애가 있어서 긴급 수정을 하였는데 그 이전에 실행하셨다면 위와 같은 오류가 생겼을 것 같습니다.🥲

**Reply by YNyum** [reply]
네...3시 이후로 유니티 재설치와 방금 전 패키지도 삭제 후 재설치하여 임포트하였는데 템플릿 생성시에 오류납니다.
패키지 임포트단계까지는 이상없었습니다.

**Reply by Zepeto-TechEvangelist** [reply]
제보 감사드립니다. 확인해 보겠습니다.

**Comment by njngwn** [comment]
비슷한 이슈를 방금 겪었는데, 그냥 유니티 파일 실행하고, package manager에서 업데이트를 했더니 해결되었습니다. 참고하시면 좋을 것 같습니다

**Reply by YNyum** [reply]
package manager에서 제페토 패키지를 업데이트 시키신건가요?

**Reply by njngwn** [reply]
네 맞습니다. 제페토 패키지 버전이 1.0.0인가 제일 낮은 걸로 되어 있더라구요. 저도 오늘 웹페이지에서 다운 받았는데, 버전이 낮게 되어 있어서 따로 제가 업데이트를 시켰더니 저는 더이상 저런 메시지가 안떴습니다!

**Reply by YNyum** [reply]
ZEPETO.World 유니티 패키지를 임포트하면 Complete 뜬 후에 패키지 안에 들어간 제페토 스크립트는 사라지는데 정상인가요?
패키지 매니저에서 안뜹니다...

**Reply by YNyum** [reply]
말씀하신게 맞았습니다. 감사합니다.
템플릿으로 열고 오류나더라도 재시도나 끄지 마시고 Continue 눌러서 열어버린 다음에 Window -> Package manager에서 업데이트 했더니 되었습니다.
혹시 같은 이슈 겪으시는 분들도 이렇게 해보세요~
